README.md

```diff
@@ -1 +1,62 @@
-# Awesome-Books-with-ES6
\ No newline at end of file
+
+# Project Name
+- Awesome Book 
+
+> A website for storing books.
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Live Demo
+
+[Live Demo Link](https://eaesmat.github.io/Awesome-Books-with-ES6/)
+
+
+## Getting Started
+
+You can get started by cloning the project or vist the demo link
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+Have Git and GitHub
+### Setup
+Copy HTTPS link of the repo and clone it in your local machine.
+### Install
+Have a code editor to make changes.
+### Usage
+This project for  anyone to use.
+### Run tests
+Run the linter before pushing the project.
+### Deployment
+This is already deployet just checkout the demo link
```

Comment:
- [ ] Kindly include the code also. For ex:-
`git clone https://github.com/eaesmat/Awesome-Books-with-ES6.git`